kubernetes/charts/weblogic-operator/values.yaml

```diff
@@ -121,6 +121,13 @@ elasticSearchPort: 9200
 # Valid values are: "SEVERE", "WARNING", "INFO", "CONFIG", "FINE", "FINER", and "FINEST".
 javaLoggingLevel: "INFO"
 
+# javaLoggingFileSizeLimit specifies the maximum size in bytes of the Java logging file.
```

Comment:
I searched the documentation using `find . -name "*.md" | xargs grep "/logs/" | grep -v tmp | grep -v shared | grep -v scratch`, and it looks like we don't have any mention of the java logging file.  

Should the doc be updated to provide context? 

If not, should it minimally be mentioned right here in the comment?  Something like "... of the Operator's logging files in its container's internal /logs/ directory"

Reply:
Or maybe do both?  The doc location I'm thinking about is `./docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md`

Reply:
I personally prefer just updating the comment, since this feels like an internal feature.

Reply:
I'll update the comment here, but leave it otherwise internal.